google/cloud/sqlalchemy_spanner/requirements.py

```diff
@@ -68,3 +68,25 @@ def get_isolation_levels(self, _):
             dict: isolation levels description.
         """
         return {"default": "SERIALIZABLE", "supported": ["SERIALIZABLE", "AUTOCOMMIT"]}
+
+    @property
+    def precision_numerics_many_significant_digits(self):
```

Comment:
This property is not true for Spanner so we should not be setting it for open as the test will fail without overrides.

Reply:
This is supported, but till scale =  9.

Reply:
Right, but one of the example numbers is scale = 12. I worry that having this open suggests that we support arbitrary precision `NUMERIC` which is not true.

From a quick search I found [this](https://github.com/zzzeek/sqlalchemy/blob/master/test/requirements.py#L1213) so I still think we should close this property.